### PR TITLE
Calculate position of master node on bootstrap screen

### PR DESCRIPTION
**Pull request by kravciak**
Needle: https://gitlab.suse.de/openqa/os-autoinst-needles-sles/merge_requests/390
Local run: http://dhcp91.suse.cz/tests/5479#step/stack_controller/25

### CODE REVIEW

File: tests/casp/stack_controller.pm [hunk 1]
```diff
@@ -70,15 +70,17 @@ sub velum_bootstrap {
...
+
     # Select master and bootstrap
-    send_key "ctrl-f";
-    type_string "master.op\n";
-    send_key "esc";
-    send_key "tab";
-    send_key "spc";
-    assert_screen "master-selected";
+    mouse_set $x, $y;
+    mouse_click;
```
Comment (coolo) [inline tests/casp/stack_controller.pm]: you might want a mouse_hide afterwards

Comment (kravciak) [inline tests/casp/stack_controller.pm]: I will add - but it disappears anyway after assert_and_click on next line (backend issue?)

Comment (kravciak) [inline tests/casp/stack_controller.pm]: done

Comment (coolo) [inline tests/casp/stack_controller.pm]: assert_and_click is actually supposed to return the mouse to the last mouse_set - which would be the checkbox

**Pull request merged**